web/app/js/components/CallToAction.jsx

```diff
@@ -1,3 +1,4 @@
+import { instructions } from './util/Utils.js';
```

Comment:
would call this variable something more descriptive than `instructions` e..g. incompleteMeshMessage but we can save this for a follow up